### PR TITLE
allow scaling of memory and cpu for boxes that define own values

### PR DESCRIPTION
**Pull request by evgeni**
useful for setups where you want to scale up the pre-defined boxes
without touching base_boxes.yaml

### CODE REVIEW

**Comment by ekohl** [comment]
I'm not sure how much sense this makes compared the complexity it adds. Is this still needed after https://github.com/theforeman/forklift/pull/485 has been merged?

**Comment by evgeni** [comment]
So the problem I want to solve is that all the pipeline boxes in config/base_boxes.yaml define a very tight memory value. I have enough RAM and want to use it, but without modifying base_boxes.yaml. and the value from settings is ignored as it gets overridden by the box.

File: README.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ Some settings can be customized for the entirety of the deployment, they are:
 
  * memory: Memory to give boxes by default unless specified by a box
  * cpus: Number of CPUs to give boxes by default unless specified by a box
+ * scale_memory: Factor to scale memory of boxes that specify an own value
+ * scale_cpus: Factor to scale CPUs of boxes that specify an own value
```
Comment (sean797) [inline README.md]: Factor to multiply... is clearer IMO.

Comment (evgeni) [reply README.md]: done

**Comment by ekohl** [comment]
Thanks!

**Comment by ekohl** [comment]
Rubocop failures were unrelated.

**Pull request merged**